dist/powerbi-client.d.ts

```diff
@@ -87,8 +87,8 @@ declare module "embed" {
     import * as models from 'powerbi-models';
     global  {
         interface Document {
-            mozCancelFullScreen: Function;
-            msExitFullscreen: Function;
+            mozCancelFullScreen: any;
+            msExitFullscreen: any;
```

Comment:
please change the definition in ts file within the src folder.
This file is auto-generated.

https://github.com/microsoft/PowerBI-JavaScript/search?q=mozCancelFullScreen&unscoped_q=mozCancelFullScreen

Reply:
We will introduce these changes in new PowerBI-Client version. we will need to close this PR because the right change is in the .ts file and not the .d.ts file.